librz/util/subprocess.c

```diff
@@ -764,7 +764,7 @@ RZ_API RzSubprocess *rz_subprocess_start_opt(RzSubprocessOpt *opt) {
 		}
 
 		// Use the previously created environment
-		environ = child_env;
+		rz_sys_set_environ(child_env);
```

Comment:
I think the problem is that this function does not really set `environ`, but it just sets a `static char **env` var defined in sys.c.

Reply:
Good point. I fixed that function, please check https://github.com/rizinorg/rizin/pull/953/commits/ecb5fd364de2d0642ab1b264e8d45c92a6255325